avocado/core/test.py

```diff
@@ -1093,13 +1094,40 @@ def fetch_asset(self, name, asset_hash=None, algorithm=None,
         :param locations: list of URLs from where the asset can be
                           fetched (optional)
         :param expire: time for the asset to expire
-        :raise EnvironmentError: When it fails to fetch the asset
-        :returns: asset file local path
+        :param find_only: whether it should only look for the asset in the
+                          cache and do not download/move it to the cache.
```

Comment:
I *think* this should read "whether it should only look for the asset in the cache and not download/move it to the cache.", that is, without the "do".  Not 100% sure though.

Reply:
I could not find a general rule about it, but if using, the correct would be _does_ and not _do_ as it refers to _it_. Anyway, I changed the wording to: 
_When `True`, `fetch_asset` only looks for the asset in the cache, avoiding the download/move action. Defaults to `False`._